app/models/setting.rb

```diff
@@ -184,7 +184,12 @@ def defaults
         "remote_census.response.gender": "",
         "remote_census.response.name": "",
         "remote_census.response.surname": "",
-        "remote_census.response.valid": ""
+        "remote_census.response.valid": "",
+        "sdg.process_debates": false,
```

Comment:
The name "process_debates" in English somehow evokes an order in my mind to process the debates :thinking:. How about `sdg.processes.debates` or `sdg.process.debates`? The former sounds better to me, but the latter is more consistent with other settings like `process.debates`, so I'd probably go with `sdg.process.debates`.